src/components/Table/Table.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {
+  getCoreRowModel,
+  getSortedRowModel,
+  Row,
+  SortingState,
+  useReactTable,
+} from '@tanstack/react-table';
+import { Flex, ScrollView, FlatList, Box } from 'native-base';
+
+import { createTableColumns } from './createTableColumns';
+import { TableHeader } from './TableHeader';
+import { TableRow } from './TableRow';
+
+type Props = {
+  data: object[];
+  onRowClick: (row: Row<any>) => void;
+  minColumnWidth?: number;
+};
+
+export const Table: React.FC<Props> = ({
+  data,
+  onRowClick,
+  minColumnWidth,
+}) => {
+  const columns = createTableColumns(data);
+  const minWidth = minColumnWidth || 80;
```

Comment:
This could just be a default prop value and this line removed

Reply:
of course, I knew I was missing something :sweat_smile: